src/views/Country/index.tsx

```diff
@@ -1,17 +1,205 @@
+    });
+
+    const {
+        pending: aggregatedAppealPending,
+        response: aggregatedAppealResponse,
+    } = useRequest<AggregatedAppealResponse>({
+        skip: !countryId,
+        url: 'api/v2/appeal/aggregated/',
+        query: { country: countryId },
+    });
+
+    const pending = countryResponsePending || aggregatedAppealPending;
 
     return (
         <Page
+            className={styles.country}
             title={strings.countryPageTitle}
+            heading={countryResponse?.name ?? '--'}
+            infoContainerClassName={styles.keyFigureList}
+            info={(
+                <>
+                    {pending && <BlockLoading />}
+                    {!pending && aggregatedAppealResponse && (
+                        <>
+                            <KeyFigure
+                                icon={<DrefIcon />}
+                                className={styles.keyFigure}
+                                value={aggregatedAppealResponse.active_drefs}
+                                description={strings.countryKeyFiguresActiveDrefs}
+                            />
+                            <KeyFigure
+                                icon={<AppealsIcon />}
+                                className={styles.keyFigure}
+                                value={aggregatedAppealResponse.active_appeals}
+                                description={strings.countryKeyFiguresActiveAppeals}
+                            />
+                            <KeyFigure
+                                icon={<FundingIcon />}
+                                className={styles.keyFigure}
+                                value={aggregatedAppealResponse.amount_requested_dref_included}
+                                normalize
+                                description={strings.countryKeyFiguresBudget}
+                            />
+                            <KeyFigure
+                                icon={<FundingCoverageIcon />}
+                                className={styles.keyFigure}
+                                value={aggregatedAppealResponse.amount_funded}
+                                normalize
+                                description={strings.countryKeyFiguresAppealsFunding}
+                            />
+                            <KeyFigure
+                                icon={<TargetedPopulationIcon />}
+                                className={styles.keyFigure}
+                                value={aggregatedAppealResponse.target_population}
+                                normalize
+                                description={strings.countryKeyFiguresTargetPop}
+                            />
+                            {countryResponse?.has_country_plan && (
+                                <KeyFigure
+                                    icon={<AppealsTwoIcon />}
+                                    className={styles.keyFigure}
+                                    value={1}
```

Comment:
just to double check if this is dummy or correct

Reply:
This is correct, if there is a country plan, we show we have one country plan